dotCMS/src/main/java/com/dotcms/util/PaginationUtil.java

```diff
@@ -224,6 +224,8 @@ private static String getUrl(final String urlBase, final String filter, final in
 
 		if (page != -1){
 			params.put(PAGE, String.valueOf(page));
+		}else{
```

Comment:
you could use params.put (PAGE, (-1 != page)?String.valueOf(page):"pageValue");
In addition pageValue could be a constant